examples/complete/dns_records.txt

```diff
@@ -0,0 +1,8 @@
+;; AAAA Records
+test-ex.aaaa.example.dev.cloud.ibm.com.	900	IN	AAAA	2001::4
+
+;; A Records
+test-ex.example.dev.cloud.ibm.com.	1	IN	A	1.2.3.4
+
+;; CNAME Records
+test-ex.example.dev.cloud.ibm.com.      900       IN      CNAME   domain.com
```

Comment:
1. Why A and CNAME records has same name?
2. A records and the one added in test - are pointing to same IP 1.2.3.4. Please make it different.
3. After adding CNAME it shows on dashboard as:
`CNAME	test-ex	is an alias of domain.com.example.dev.cloud.ibm.com`
domain.com.<actual domain given in example> --> seems to be incorrect. It needs an update.